packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -49,12 +68,17 @@ export default function Sidebar({
   } = useContext(SettingsContext);
   const { modal, openModal } = useLazyModal();
   const showSettings = modal?.type === LazyModal.UserSettings;
+  const { isNewMobileLayout } = useMobileUxExperiment();
+  const isTablet = useViewSize(ViewSize.Tablet);
 
   const feedName = getFeedName(activePageProp, {
     hasUser: !!user,
     hasFiltered: !alerts?.filter,
   });
   const activePage = `/${feedName}`;
+  const isMyFeed =
+    router.pathname === '/' || feedName === SharedFeedPage.MyFeed;
+  const isProfilePage = router.pathname?.includes('/[userId]');
```

Comment:
Don't think this is particularly right.
'/' can be any of your feeds that you last visited basically (popular/myfeed/discussion etc)

Maybe check how we do it in other sidebar, this logic should exist for setting active.

Reply:
good point, upon a second look, the pathname should not matter, as the `feedName` is already handling everything under the hood. I'll clean this up 🙏